maintainers/maintainer-list.nix

```diff
@@ -10244,6 +10244,11 @@
     githubId = 691290;
     name = "Keshav Kini";
   };
+  kintrix = {
```

Comment:
An e-mail is required.
If not cluttering your mailbox is a concern, I would suggest a Protonmail.